app/views/shared/_event-modal.html.erb

```diff
@@ -0,0 +1,26 @@
+<% if @event %>
+  <div class="row">
+    <div class="small-12 columns">
+      <div class="c-events-modal">
+        <div class="event-container" style="background-image: url(<%= asset_path 'demo-event-bg.png' %>);">
+          <div class="js-btn-events-modal">
+            <svg class="icon icon-share">
+              <use xlink:href="#icon-close"></use>
+            </svg>
+          </div>
+          <div class="event-info">
+            <h3 class="text -title-single-alt"><%= @event.title %></h3>
+            <p class="text -body-single"><%= @event.description %></p>
```

Comment:
Make sure to parse the text on the description, as we are using a WYSIWYG on the back!